File: android/app/src/test/java/com/purchasely/shaker/TestHelpers.kt
package com.purchasely.shaker

import com.purchasely.shaker.domain.model.Cocktail
import com.purchasely.shaker.domain.model.Ingredient

fun testCocktail(
    id: String = "1",
    name: String = "Mojito",
    spirit: String = "Rum",
    category: String = "Classic",
    difficulty: String = "Easy",
    tags: List<String> = listOf("test"),
) = Cocktail(
    id = id,
    name = name,
    image = "$id.jpg",
    description = "A test cocktail",
    category = category,
    spirit = spirit,
    difficulty = difficulty,
    tags = tags,
    ingredients = listOf(Ingredient("Ingredient", "60ml")),
    instructions = listOf("Mix and serve")
)

File: ios/ShakerTests/Mocks/TestHelpers.swift
@testable import Shaker

func testCocktail(
    id: String = "1",
    name: String = "Mojito",
    spirit: String = "Rum",
    category: String = "Classic",
    difficulty: String = "Easy"
) -> Cocktail {
    Cocktail(
        id: id,
        name: name,
        image: "\(id).jpg",
        description: "A test cocktail",
        category: category,
        spirit: spirit,
        difficulty: difficulty,
        tags: ["test"],
        ingredients: [Ingredient(name: "Ingredient", amount: "60ml")],
        instructions: ["Mix and serve"]
    )
}
